lib/src/main/java/org/openapitools/openapistylevalidator/OpenApiSpecStyleValidator.java

```diff
@@ -240,17 +240,45 @@ private void validateNaming() {
                                                         method);
                                             }
                                         } else {
-                                            isValid = namingValidator.isNamingValid(
-                                                    opParam.getName(), parameters.getParameterNamingConvention());
-                                            if (!isValid) {
-                                                errorAggregator.logOperationBadNaming(
-                                                        opParam.getName(),
-                                                        "parameter",
-                                                        parameters
-                                                                .getParameterNamingConvention()
-                                                                .getDesignation(),
-                                                        key,
-                                                        method);
+                                            if (opParam.getIn() == Parameter.In.QUERY) {
+                                                isValid = namingValidator.isNamingValid(
```

Comment:
for the if, and the 2 else if, there is too much copy paste to my taste here.
Please refactor by extracting the inner of each if and replace it with a f unction taking 2-3 parameters.

Reply:
Hi @JFCote,
I pushed some changes to do a small refactoring in the code and a documentation update.
Please check if everything is ok when possible.
Thanks!